app/src/main/java/io/gnosis/safe/di/components/ViewComponent.kt

```diff
@@ -73,8 +75,12 @@ interface ViewComponent {
 
     fun inject(fragment: SettingsFragment)
 
+    fun inject(fragment: OwnerHowToAddFragment)
```

Comment:
Why `OwnerHowToAddFragment` and not `OwnerAddFragment` ?

Reply:
because it's not adding an owner. This fragment informs about different possibilities how to add owner and proposes different ways to proceed.

Reply:
What about `OwnerAddOptionsFragment` ?